### ISSUE TITLE
Cannot reproduce the result

### ISSUE THREAD

**Issue by shlee625** [NVlabs/OmniDrive#74]
Hi authors, thank you for your great work! It's really interesting.

I think you're aware of this problem but it's not reproducible using out-of-box setting described in docs. I also tried to checkout to https://github.com/NVlabs/OmniDrive/commit/7dd5bb9a8200d5d7865b1227867d047f12123b93 and train the model as you suggested but the result is same.
There is an issue (https://github.com/NVlabs/OmniDrive/issues/63) talking about this problem as well. A folk shared his experience that lower learning rate (`1e-6`) could help stable training. I also tried this but the validation result at the end of the training gives almost zero mAP.

Here are the things I want to ask you:
1. It seems that the LoRA rank is set as `128` in this log you shared before but in https://github.com/NVlabs/OmniDrive/commit/7dd5bb9a8200d5d7865b1227867d047f12123b93 it is set as `16`. Did you train this model in the older commit?
2. In the training log you shared, it seems that PyTorch is not the prebuilt one (`PyTorch: 1.13.0a0+936e930`). Did you build PyTorch on your own?
3. Is there any progress regrading this issue?

It would be really grateful if you can help with this reproduction issue. Thank you.

**Comment by exiawsh** [comment]
I use nvidia ngc pytorch docker.
I try both lora rank 128 and 16 before. Both setting works well. May I ask the version of your PEFT and which pkl files did you use?

**Comment by shlee625** [comment]
I used the ones provided by you. I didn't do anything but following the setup instruction regarding to the data.
